xl-op/blueprint.yaml

```diff
@@ -708,6 +708,180 @@ spec:
       prompt: "Provide storage class for the server:"
       promptIf: !expr ServerType != 'dai-release-runner' && ProcessType == "install"
       description: Provide storage class for the server
+    - name: ResourcesSource
+      type: Select
+      options:
+        - label: Predefined resource values
+          value: preset
+        - label: Custom resource values
+          value: editor
+      saveInXlvals: true
+      ignoreIfSkipped: true
+      overrideDefault: true
+      prompt: "Select source of the resource values:"
+      promptIf: !expr ServerType != 'dai-release-runner' && ProcessType == "install"
+      description: "Provide source of the resources requests and limits values for various containers"
+      default: editor
+    - name: ResourcesPresetDeploy
+      type: Select
+      options:
+        - label: request is cpu 1.0 and memory 2Gi. limit is 150% of request
+          value: nano
+        - label: request is cpu 2.0 and memory 4Gi. limit is 150% of request
+          value: micro
+        - label: request is cpu 4.0 and memory 8Gi. limit is 150% of request
+          value: small
+        - label: request is cpu 8.0 and memory 16Gi. limit is 150% of request
+          value: medium
+        - label: request is cpu 16.0 and memory 32Gi. limit is 150% of request
+          value: large
+        - label: request is cpu 32 and memory 64Gi. limit is 150% of request
+          value: xlarge
+        - label: request is cpu 64 and memory 128Gi. limit is 150% of request
+          value: 2xlarge                                                                      
```

Comment:
The values should change according to the changes in the chart impl.

Reply:
Support for different settings on deploy and release. It will be split for sure at some point.